client/state/data-layer/wpcom/me/devices/index.js

```diff
@@ -15,23 +15,44 @@ import {
 import { userDevicesAdd } from 'state/user-devices/actions';
 import { errorNotice } from 'state/notices/actions';
 
-export const requestUserDevices = function( { dispatch } ) {
-	dispatch( http( {
-		apiVersion: '1.1',
-		method: 'GET',
-		path: '/notifications/devices',
-	} ) );
-};
+const devicesFromApi = devices => keyBy( devices.map(
+	( { device_id, device_name } ) => ( { id: device_id, name: device_name } )
+), 'id' );
```

Comment:
I did not mean to imply that we required a separate `fromAPI` function, though I _do_ encourage it.